src/core/function/config.js

```diff
@@ -49,23 +52,33 @@ export function configFactory (config, emit) {
    */
   function _config (options) {
     if (options) {
-      const prev = mapObject(config, clone)
+      if (options.epsilon !== undefined) {
+        // backwards compatibility
+        console.warn('Warning: The configuration option "epsilon" is deprecated. Use "relTol" and "absTol" instead.')
+        const optionsFix = mapObject(options, clone)
```

Comment:
I think this can be simplified to `const optionsFix = clone(options)`

Reply:
O that's right, it works also for all instances of mapObject, so mapObject could be removed from importrs.